src/App.css

```diff
@@ -1,39 +1,59 @@
-.App {
```

Comment:
- [ ] As required for the project, kindly make sure all the boilerplate that got created by the CRA (texts, images, and styles) should be deleted. Right now, I can that you still have files non related files created by CRA such as `src/reportWebVitals.js`, `src/setupTests.js`, `App.test.js`,... I suggest you delete them